include/libdnf5/utils/bgettext/bgettext-mark-domain.h

```diff
@@ -26,6 +26,9 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 
 #include "bgettext-mark-common.h"
 
+// Use EMPTY_MESSAGE instead of M_(""). Otherwise, xgettext breaks catalogs.
+#define EMPTY_MESSAGE \
+    { .bgettextMsg = "\004" GETTEXT_DOMAIN "\00" "" }
```

Comment:
Can you please fix the formatting?  See https://github.com/rpm-software-management/dnf5/actions/runs/7503905055/job/20429713582?pr=1162

Reply:
Do you mean suppress clang-format, or format all the macros according to the clang-format's taste? I've just pushed a change which does the former.

Reply:
Thanks, I also find suppressing clang format better here.